### PR TITLE
Endoc 414C new PR with cleaned up branch

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/.vuepress/next.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
                 title: 'Introduction',
                 children: [
                     {
-                        title: 'Overview',
+                        title: 'Entando Platform',
```
Comment (nshaw) [inline vuepress/docs/.vuepress/next.js:8]: I think we should keep it simpler (switching #3 and #4).
- Overview
- Getting Started
- Solution Template
- Development Process
- Entando Architecture

It looks like we need to add Solution Templates to 6.3.2?

**Review by nshaw — CHANGES_REQUESTED** [review]

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+Business IT groups and developers Compose an Application with the Entando App Builder, a low-code composition UI. Assemble or update an application from the components already built in step one, adding templates, widgets, or full PBCs for functionality. The Entando Component Manager and Repository are integrated into the App Builder, making the composable process flexible and agile by managing all your components there.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:37]: agree with leading with target user, as is done in revised Curate and Share description

would change Business IT groups to Business units; BU is a common acronym used in this context

should use full name of Entando Component Repository for consistent naming convention and to clarify that the Manager and Repository are two separate entities; also add links

sample rephrasing: "Business units and developers Compose an application with the Entando App Builder, a low-code composition UI. Assemble or update an application from pre-built components, including templates, widgets and PBCs to add functionality. The Entando Component Manager and Entando Component Repository are integrated into the App Builder for flexible and agile component management."

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder]<!--(../compose/app-builder.md)--> | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager]<!--(../compose/ecm-overview.md)--> | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume Applications 
+In the Consume step, the end-user or consumer interacts with an application to complete specific tasks, the runtime phase of an Entando Application. The modular design of the platform allows the application and services to be scaled according to that use. In the same way, applications can be continuously improved by adding or updating web content, changing engine rules, and deploying seamless upgrades to individual components.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:48]: how is an end user different from a consumer? should be obvious the user is the one consuming. would remove hyphen and "consumer"

first sentence is awkward... not explicit if consumer interaction or task completion is the runtime phase; don't think mention of runtime phase adds value... nothing unique to Entando with respect to runtime and the concept of runtime isn't relevant here

sample rewording: "The end user Consumes content by interacting with an application to complete specific tasks. The modular design of the Entando Platform enables intelligent scaling of applications and services. This modularity also allows applications to be iteratively improved by adding or updating content, modifying engine operation, and deploying seamless upgrades to individual components."

**Review by Lyd1aCla1r3 — COMMENTED** [review]
apologies once again for the extensive feedback... take it or leave it!

please me know if you disagree with anything i suggested; that will influence my writing moving forward

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+# Development Concepts on Entando
+ 
+With the Entando Platform, the development of an application is organized into distinct steps: create, curate, compose, and consume. In the simplest terms, you start by **creating** functional building blocks, **curate** the blocks in a catalog, **compose** them on a page or website, which is then **consumed** in an app where it can be scaled and improved upon. 
+
+Entando provides a composable environment where these building blocks can be built independently, enabling parallel teams to accelerate development and innovation on a cloud and Kubernetes-native stack.
+
+![Entando Platform Development Process](./img/concepts.png)
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independant component or functional block for your application. Developers can use the Entando Component Generator or Blueprint to automate the building and packaging of project files, with the flexibility to customize. In this way, new business capabilities can be integrated into your application without rewriting existing systems.
```
[the remark this answers is not in the document]
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/getting-started/development-concepts.md:14]: allowing new business capabilities to integrate is equivalent to saying they can be integrated... neither phrasing impacts other components and the optional language is preserved

i would still use Entando Blueprint... only one instance of entando in that paragraph and it's important to clarify that the repo and blueprint are separate entities. if you include links to each that helps, but you're introducing an entando-branded thing, so should make that clear. just my 2 cents. again :P

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+sidebarDepth: 2
+---
+
+# Development Concepts on Entando
+ 
+With the Entando Platform, the development of an application is organized into distinct steps: create, curate, compose, and consume. In the simplest terms, you start by **creating** functional building blocks, **curate** the blocks in a catalog, **compose** them on a page or website, which is then **consumed** in an app where it can be scaled and improved upon.
```
[the remark this answers is not in the document]
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/getting-started/development-concepts.md:7]: i can get behind that, too. but would be consistent with "you" or a particular user role.

File: vuepress/docs/.vuepress/next.js (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/.vuepress/next.js:8]: @jyunmitch I don't think these changes made it in. I'm adding a PR with that and a couple other suggestions.

Comment (nshaw) [reply vuepress/docs/.vuepress/next.js:8]: https://github.com/entando/entando-docs/pull/417/files

**Review by nshaw — CHANGES_REQUESTED** [review]

**Review by nshaw — APPROVED** [review]

File: vuepress/docs/next/docs/getting-started/development-concepts.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder](../compose/app-builder.md) | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager](../compose/ecm-overview.md) | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume Applications 
+In the development lifecycle of an application, Consume is the execution stage where end users interact with the app to complete certain tasks. For an Entando application, the modular design of the components and services means each can be scaled individually or as a whole, according to those interactions. Likewise, content can be added or changed, engine rules adapted and upgrades made seamlessly. Thus the development cycle is a continuous process with minimal disruptions and faster time to market.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/getting-started/development-concepts.md:49]: "In" an Entando Application, not "for"

Entando Application -> capitalize

same comment wrt "means"
tagging ".., according to those interactions" on the end is awkward

what are upgrades outside of adding/editing content and adapting engine rules? if upgrades fall within those categories mentioning upgrades is redundant. seamless upgrades comprise improvements to content and engine rules 

use of "thus" is awkward

**Review by Lyd1aCla1r3 — COMMENTED** [review]
i really liked your choice of "adapt" wrt engine rules, and how you clarified that Consume is the execution stage - not limited to end user actions. that encompasses runtime + users without being overly wordy and explicit

**Pull request merged**